autoortho/getortho.py

```diff
@@ -688,18 +691,20 @@ def get_img(self, mipmap, startrow=0, endrow=None, maxwait=5, min_zoom=None):
         #
 
         # Get effective zoom
-        zoom = self.zoom - mipmap
+        gzo_effective = min(self.global_zoom_out, max(0, 4 - mipmap))
```

Comment:
The _get_quick_zoom function already centalizes and returns an adjusted zoom and mipmap level, should this calculation be there instead?